appyx-interactions/android/src/main/kotlin/com/bumble/appyx/interactions/sample/TestDriveExperiment.kt

```diff
@@ -84,12 +81,16 @@ fun TestDriveExperiment() {
     Column(
         Modifier
             .fillMaxWidth()
-            .background(appyx_dark)
+            .background(Color.Black),
```

Comment:
Why?

Reply:
`appyx_dark` wasn't defined in js scope as it's part of Android theme.